### ISSUE TITLE
Search Results: Visually Indistinct “Current Filters” Heading

### ISSUE THREAD

**Issue by jscholes** [benetech/ImageShare#59]
The “Current Filters:” heading text, appearing before the currently applied filter buttons, is not visually distinct and should at least appear in bold.

![Screenshot of the text “Current Filters:” appearing before the “heart” search filter and its associated remove button.  The heading text has the same styling as the surrounding page text.](https://user-images.githubusercontent.com/6454755/80003779-1e2df080-8487-11ea-936d-b1a118b92ca3.png)

**Comment by clapierre** [comment]
Fine to make that a lot more distinct, do you think it needs to be some sort of accessibility landmark / heading?

**Comment by jscholes** [comment]
> do you think it needs to be some sort of accessibility landmark / heading?

Definitely a heading, and maybe a labelled region as well.  If we have heading styling wich applies across the site, this can probably just fit into that?

**Comment by clapierre** [comment]
Sounds like a plan.  Lets come up with a consistent approach to how we do landmarked regions and headings so we are all on the same page and it is consistent across all our pages.

**Comment by sinabahram** [comment]
one suggestion on this.

 

We’ll make sure to map this to nice semantic markup when it comes time for @jkva to implement, especially since @jscholes is on the case in terms of semantics. Let’s concentrate our high-level discussions around where things should be/how they should behave, an then let’s have a separate conversation, if needed, about markup choices.

 

I only say this because as the designs change, markup choices may change too, as they should, so it’s too soon to think markup, I feel.

**Comment by jscholes** [comment]
Agreed.

**Comment by sinabahram** [comment]
This has all been resolved by new designs, I believe.
